internal/provider/resource_managed_identity_access_rules.go

```diff
@@ -153,6 +156,15 @@ func (t *managedIdentityAccessRuleResource) Schema(_ context.Context, _ resource
 				},
 				// Can be updated in place, so no RequiresReplace plan modifier.
 			},
+			"verify_state_lineage": schema.BoolAttribute{
+				MarkdownDescription: "Whether to verify the state lineage's attestation status, default is false.",
+				Description:         "Whether to verify the state lineage's attestation status, default is false.",
+				Optional:            true,
+				Computed:            true, // When not passed it, it needs to be set by Create.
+				PlanModifiers: []planmodifier.Bool{
+					boolplanmodifier.RequiresReplace(),
```

Comment:
This should not require a replace since the field can be updated w/o replacing the rule.